lib/peep.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'database_connection'
+
+class Peep 
+  attr_reader :id, :message, :timestamp, :user_id
+  def initialize(id:,message:,timestamp:,user_id:)
+    @id = id
+    @message = message
+    @timestamp = timestamp
+    @user_id = user_id
+  end
+
+  def self.create(message:, user_id:)
+    current_time = Time.now.strftime("%Y-%m-%d %H:%M:%S")
+    result = DatabaseConnection.query(
+      "INSERT INTO peeps(message,timestamp,user_id) VALUES($1,$2,$3) RETURNING id, message, timestamp, user_id;", 
```

Comment:
in this case, what do $2 and $3 refer to?

Reply:
These are placeholders, they only get filled with the actual values (stored in the array outside of the sql code). The idea being that this prevents any SQL injection attacks!